src/explore-education-statistics-frontend/src/modules/find-statistics/components/SortControls.tsx

```diff
@@ -0,0 +1,66 @@
+import { Form, FormRadioGroup, FormSelect } from '@common/components/form';
+import { useMobileMedia } from '@common/hooks/useMedia';
+import { PublicationSortOptions } from '@common/services/publicationService';
+import { Formik } from 'formik';
+import noop from 'lodash/noop';
+import React from 'react';
+
+const options = [
+  { label: 'Newest', value: 'newest' },
+  { label: 'Oldest', value: 'oldest' },
+  { label: 'A to Z', value: 'alphabetical' },
+];
+
+interface FormValues {
+  sortBy: PublicationSortOptions;
+}
+
+interface Props {
+  initialValues: FormValues;
```

Comment:
Don't need to do this as `initialValues` as this isn't a particularly complicated form (we're just concerned with sorting). Could just use a `sortBy` prop to simplify things.